app/policies/user_policy.rb

```diff
@@ -25,6 +25,14 @@ def update_supervisor_name?
     update_supervisor_email?
   end
 
+  def update_user_setting?
+    if is_supervisor?
+      # allow access to own record or volunteer record
+      return record == user || record.volunteer?
```

Comment:
permissions issue- must make sure that the volunteer belongs to the correct casa org

Reply:
@compwron I have a question. Where is this volunteer? function defined for "record.volunteer?".

Reply:
done